### PR TITLE
RoughTime breaks some specs

### PR DESCRIPTION
**Pull request by spuun**

### CODE REVIEW

File: src/lavinmq/rough_time.cr [hunk 1]
```diff
@@ -1,26 +1,43 @@
 module RoughTime
-  @@utc = Time.utc
-  @@unix_ms : Int64 = @@utc.to_unix_ms // 100 * 100
-  @@monotonic = Time.monotonic
+  {% if flag?(:realtime) %}
+    @[AlwaysInline]
```
Comment (carlhoerberg) [inline src/lavinmq/rough_time.cr:3]: I'm pretty sure the compiler will figure that out without the attribute

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -135,7 +135,7 @@ jobs:
       - name: Run LavinMQ in background
         run: |
           chmod +x bin/*
-          bin/lavinmq --data-dir=/tmp/amqp --bind=:: &
+          bin/lavinmq --data-dir=/tmp/amqp --bind=:: --log-level=debug &
```
Comment (carlhoerberg) [inline .github/workflows/ci.yml:138]: left over?

File: src/lavinmq/rough_time.cr (hunk 1 above)
Comment (spuun) [reply src/lavinmq/rough_time.cr:3]: Yes, most likely.

**Comment by spuun** [comment]
Closing this for now. We're handling some test by modifying other java client tests.

**Pull request closed without merging**